examples/features/torch_batch_process/batch_inference/compare_with_core_api/README.md

```diff
@@ -0,0 +1,24 @@
+# Batch inference with Core API & Torch Batch Processing API
```

Comment:
Does this example need to be so deep in folders? Couldn't we move it if we add more layers? It seems weird to go down 3 folders with nothing in them for a single example

Reply:
I will be adding another folder on the same level as compare_with_core_api in another PR.

Reply:
I agree with taylor, I think this will end up being two entire layers of hierarchy deeper than its peers in `examples/features/*`.

Is there a way we can rejigger things to match the existing `examples/features/*` hierarchy?  Maybe something like:
- `examples/features/torch_batch_inference`  (this example)
- `examples/features/torch_generate_embeddings`  (the bert embeddings example)

Reply:
Moved to examples/features/torch_batch_process_core_api_comparison